roles/sudoers/templates/92-ansible-managed

```diff
@@ -4,6 +4,6 @@
 #
 # {{ ansible_managed }}
 #
-# Allow specific users or group to become the {{ item.user }} user.
+# Allow specific users or group to become the {{ item.name }} user.
```

Comment:
This comment no longer makes sense. As we now not only allow sudo to a functional account, but also allow users to run specific tasks with sudo, the `item.name` may be a tool/command like `apptainer`. Please rephrase.